xbundle/__init__.py

```diff
@@ -21,10 +21,10 @@
 import string
 import glob
 import subprocess
+from os.path import join, exists
 
 from lxml import etree
 from lxml.html.soupparser import fromstring as fsbs
-from path import path	# needs path.py
```

Comment:
Any particular reason for removing [path.py](https://pypi.python.org/pypi/path.py) instead of just adding it to deps?  Also, a change like that might be nicer as a separate PR, or at least one discrete commit in the future.